lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.html

```diff
@@ -2,12 +2,27 @@
 {% load i18n %}
 {% load django_markup %}
 {% load static %}
+{% block table_style %}
```

Comment:
I was previously using negative margins to create a full-width banner but it looks email clients don't support negative margins so I had to override the parent template css from the child template